templates/layout.html

```diff
@@ -1,9 +1,20 @@
 <!doctype html>
 <title>Treebank Annotation</title>
 <script src="/annotate/static/script.js"></script>
+<!-- Include jQuery -->
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+<!-- Include jQuery UI -->
+<link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"></script>
 <link rel=stylesheet type=text/css href="/annotate/static/style.css">
 {%- for message in get_flashed_messages() %}
 <div class=flash>{{ message }}</div>
 {% endfor -%}
-
+<div id="sentEntry" title="Enter Sentence" style="display:none;">
+    <p>Please enter the sentence to be added:</p>
+    <input style = "width:100%" type="text" id="sentenceInput" />
+    <p>Enter a unique id for the sentence:</p>
+    <input type="text" id="idInput" /> <p>
```

Comment:
I can imagine a workflow where the user doesn't really care about the sentence ID so long as it is unique. Is it possible to show a default value, e.g. `id#` where `#` is the number of sentences created so far plus 1?

Reply:
Hash as the default ID implemented in 06aedc7